src/components/link/Link.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+import LinkBase, { LinkBaseProps } from './LinkBase';
+
+export interface LinkProps extends LinkBaseProps {
+    children: React.ReactNode;
+    className: string;
+}
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
I wonder why this is not stateless, can just default className = ''.
Anyways maybe add it to the notes section of the box note (or maybe we add a todo column), so that we can come back and clean this up.

Reply:
I will clean up these components in this PR

Reply:
I tried converting `Link` and `LinkGroup` to stateless components, but that caused a bunch of failures in snapshot tests for other components. I added a couple of todo comments and will continue investigating in a separate PR.